regression/goto-instrument/nondet_static/main.c

```diff
@@ -0,0 +1,8 @@
+const int g_A = 100;
+const int g_B[]={10, 20, 30};
+int g_var;
```

Comment:
To what extent are `g_A` and `g_var` relevant to this test (or, rather, the fact that they are global)?